pyfiglet/__init__.py

```diff
@@ -101,13 +102,18 @@ def preloadFont(cls, font):
                 data = pkg_resources.resource_string('pyfiglet.fonts', fn)
                 data = data.decode('UTF-8', 'replace')
                 return data
+            elif os.path.isfile(font):
+                with open(font, 'rb') as f:
+                    return f.read().decode('UTF-8', 'replace')
         else:
             raise FontNotFound(font)
 
     @classmethod
     def isValidFont(cls, font):
         if not font.endswith(('.flf', '.tlf')):
             return False
+        if os.path.isfile(font):
+            return True
```

Comment:
This check doesn't do the same thing as the `pkg_resources` path. Please can you refactor it so that it does the equivalent check?